server/groups.py

```diff
@@ -0,0 +1,15 @@
+import requests
+from flask import request
+
+
+def get_invites_for_account(account):
+    if account:
```

Comment:
No need to check if account is null. Account is provided as a parameter and must therefore be non-null.